football/spec/models/user_spec.rb

```diff
@@ -2,15 +2,32 @@
 
 RSpec.describe :User, type: :model do
 
-  describe "validate email" do 
-    before(:each) do
+  before(:all) do
       @user = FactoryGirl.build(:user)
     end
-    subject { @user }
+  subject { @user }
+
```

Comment:
Those lines are not needed, you can use [this site](https://www.relishapp.com/rspec/rspec-rails/docs/gettingstarted) as a references for rspec and cucumber